main.py

```diff
@@ -55,3 +56,9 @@
                                     "title-2-conservation/eqip_ira/20231106-2024_2031-EQIPextrafund-project-by-practice-MAX-clean.xls",
                                     "title-2-conservation/eqip_ira/20240702_BA_EQIP_IRA_base2023_value.xlsx")
     eqip_ira_parser.parse_and_process()
+
+    house_outlay_parser = HouseOutlayParser("2023", 2024, 2033,
```

Comment:
I think the first parameter could be passed as int. In the class constructor, it's converted to int before storing in the class attribute.